src/main/java/httpclient/helpers/ResponseHandler.java

```diff
@@ -0,0 +1,25 @@
+package httpclient.helpers;
+
+import org.apache.hc.client5.http.impl.classic.CloseableHttpResponse;
+import org.apache.hc.core5.http.ParseException;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+public class ResponseHandler {
+
+    public static JSONObject getResponseAsJSONObject(CloseableHttpResponse response) {
+        String str;
+        try {
+            str = EntityUtils.toString(response.getEntity());
+        } catch (IOException | ParseException e) {
+            throw new RuntimeException(e.getMessage());
```

Comment:
Why do you handle exceptions in such a manner?

Reply:
refactor exception handling